src/main/java/minicraft/network/MinicraftClient.java

```diff
@@ -324,17 +327,26 @@ public boolean parsePacket(InputType inType, String alldata) {
 				}
 				
 				String[] tilestrs = alldata.split(",");
-				byte[] tiledata = new byte[tilestrs.length];
-				for (int i = 0; i < tiledata.length; i++)
-					tiledata[i] = Byte.parseByte(tilestrs[i]);
+				MinicraftServer.TileDataPair[] tiledata = new MinicraftServer.TileDataPair[tilestrs.length];
+				for (int i = 0; i < tiledata.length; i++) {
+					byte l = Byte.parseByte(tilestrs[i]);
+					i++;
+					try {
+						tiledata[i] = new MinicraftServer.TileDataPair(l, NBTReader.read(new ByteArrayInputStream(tilestrs[i].getBytes())));
+					} catch (IOException e) {
+						// TODO Auto-generated catch block
+						e.printStackTrace();
+						tiledata[i] = new MinicraftServer.TileDataPair(l, new NBTCompound());
+					}
+				}
```

Comment:
This is a bit pointless considering multiplayer is disabled, and is also planned to be removed

Reply:
So it can be removed in the future since I need to resolve the changes.